components/sync15/src/changeset.rs

```diff
@@ -82,7 +92,7 @@ impl IncomingChangeset {
 #[derive(Debug, Clone)]
 pub struct CollectionUpdate<'a, 'b> {
```

Comment:
You didn't add this, but neither of these are mut so I'm pretty sure we should be able to only have one lifetime parameter.